server/service/apple_mdm.go

```diff
@@ -587,15 +587,24 @@ func additionalDigiCertValidation(contents string, digiCertVars *digiCertVarsFou
 						foundCAs = append(foundCAs, ca)
 						break
 					}
-					return &fleet.BadRequestError{Message: "CA name mismatch between $FLEET_VAR_DIGICERT_DATA_<ca_name> and $FLEET_VAR_DIGICERT_PASSWORD_<ca_name> in PKCS12 profile"}
+					payloadContent := string(payload.PayloadContent)
+					if len(payloadContent) > 100 {
```

Comment:
would prefer the magic number 100 to be a simple const like `error_payload_message_limit` otherwise this looks good